apps-rendering/src/renderer.ts

```diff
@@ -321,7 +321,18 @@ const textElement =
 			case 'SUB':
 				return h('sub', { key }, children);
 			case 'SUP':
-				return h('sup', { key }, children);
+				return styledH(
+					'sup',
+					{
+						key,
+						css: {
+							position: 'relative',
+							top: '-0.5em',
+							fontSize: '75%',
```

Comment:
Do you think it would make sense to use the rules the browsers (Safari and Firefox) use for superscript? `vertical-align` in particular has a specific keyword for this [^1].

```suggestion
							fontSize: 'smaller',
							verticalAlign: 'super',
```

[^1]: https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align#super

Reply:
Weird, thought I had but no. Just fixed that